engine/schema/src/com/cloud/user/UserAccountVO.java

```diff
@@ -33,6 +33,7 @@
 
 import com.cloud.utils.db.Encrypt;
 import com.cloud.utils.db.GenericDao;
+import com.google.common.base.Strings;
```

Comment:
Aren't we supposed to use cloud.com.utils StringUtils here?

Reply:
We've been using `Strings.isNullOrEmpty`, and com.cloud.utils.StringUtils does not provide a utility to check if a string is empty or null.